### PR TITLE
[FIX] toggle active service button

### PR DESCRIPTION
**Pull request by mymage**
I tried to use the same button present on Vehicle without success, so I used an alternative, but Travis dislike it.
Surely you can suggest a better way or say why the original button doesn't work.

### CODE REVIEW

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml [hunk 1]
```diff
@@ -16,10 +16,14 @@
...
-                        Archived
+                    >
+                    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
+                        <!--
+                            this option doesn't works, why?
+                            <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
+                            -->
+                        <label attrs="{'invisible': [('active', '!=', True)]}">Active</label>
+                        <label attrs="{'invisible': [('active', '!=', False)]}">Archived</label>
                     </button>
```
Comment (mamcode) [inline fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]:
```suggestion
                    <button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
                        <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
                    </button>
```

Comment (mamcode) [reply fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: Also remember remove the line https://github.com/OCA/fleet/blob/5ebf8d3c69a34a1d06f0b2f90155ad903506e217/fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml#L31

**Review by mamcode — CHANGES_REQUESTED** [review]
With the following suggestion of changes it should work.

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml (hunk 1 above)
Comment (ivantodorovich) [reply fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: @mymage can you check @mamcode 's suggestion out?

It should be like this:

```xml
<button name="toggle_active" type="object" class="oe_stat_button" icon="fa-archive">
    <field name="active" widget="boolean_button" options='{"terminology": "archive"}'/>
</button>
 ```

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

File: fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml (hunk 1 above)
Comment (mymage) [reply fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml]: Thanks guys: so the problem was `<field name="active" invisible="1" />`.
Locally tested and works.

**Review by mamcode — APPROVED** [review]
Good!

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**